src/main/java/org/spongepowered/mod/mixin/entity/projectile/MixinEntityArrow.java

```diff
@@ -111,4 +114,16 @@ public void setCritical(boolean critical) {
     public MixinEntityArrow(World worldIn) {
         super(worldIn);
     }
+
+    @Override
+    public void readFromNbt(NBTTagCompound compound) {
```

Comment:
@gabizou It does, [see here](https://github.com/SpongePowered/Sponge/pull/173/files#diff-d1e54a67a6e6054ffea2c372f361a87cR394)